### ISSUE TITLE
multiline prompt has insertion point at end of first line

### ISSUE THREAD

**Issue by michaelstack** [dbcli/mycli#220]
When I include newlines in my prompt, the resultant prompt looks right, but the insertion point is at the end of the first line.

For example, if I specify my prompt in the config file thus:
```
prompt = "firstline\nsecondline\n> "
```

The resultant prompt is:
```
firstline
secondline
>
```

It looks ok till I start typing. My text inserts at the end of "firstline". I would expect it to insert at the end of the third line ("> ").

**Comment by mdsrosa** [comment]
@michaelstack Well observed. Thank you for reporting this. I'll take a look and try to see what can be done to address that. But I'm not sure that this should be the default behavior. Have you tried on mysql standard client?

**Comment by amjith** [comment]
@michaelstack This is a known issue which is caused by prompt_toolkit. 

I think it is best to file an issue with them. https://github.com/jonathanslenders/python-prompt-toolkit

**Comment by amjith** [comment]
Released as part of 1.6.0. Please upgrade: 

```
pip install -U mycli 
```
